inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -233,6 +233,7 @@ DeviceName AutoInferencePlugin::SelectDevice(const std::vector<DeviceName>& meta
 
     std::vector<DeviceName> CPU;
     std::vector<DeviceName> GPU;
+    std::vector<DeviceName> VPU;
```

Comment:
better rename to "MYRIAD", also for the PR title name.

Reply:
VPUX > dGPU > iGPU > CPU > MYRIAD now